setup.py

```diff
@@ -112,7 +112,8 @@ def find_rel_test():
     zip_safe=False,
     install_requires=[
         'setuptools >= 0.6',
-        "vsc-base >= 2.4.18",
+        "vsc-base >= 2.5.0",
```

Comment:
what you specify here depends on how you bump the `vsc-base` version in https://github.com/hpcugent/easybuild-framework/pull/225, which should most likely be a minor bump, although I guess bumping to 2.5.0 wouldn't hurt anyone (it's just that there's nothing shocking being added that may warrant a bump from 2.4.x to 2.5.0...)